framework/include/task_manager/task_manager.h

```diff
@@ -133,17 +133,17 @@ typedef struct tm_msg_s tm_msg_t;
 /**
  * @brief Unicast callback function type
  */
-typedef void (*_tm_unicast_t)(tm_msg_t *);
+typedef void (*tm_unicast_callback_t)(tm_msg_t *);
```

Comment:
The set_unicast_cb API does not use function pointer. Why do you expose this type?
There are two options.
option 1. Let's move this into task_manager_internal.h. 
option 2. Let's use this type in set_unicast_cb API like below:
    `int task_manager_set_unicast_cb(tm_unicast_callback_t cb_func);`